twinkle.js

```diff
@@ -426,7 +426,7 @@ $.ajax({
 				Twinkle.prefs.optionsVersion = Twinkle.prefs.optionsVersion || 1;
 			}
 		} catch (e) {
-			mw.notify('Could not parse your Twinkle preferences', {type: 'error'});
+			mw.notify('Could not parse your Twinkle preferences');
```

Comment:
Isn't this actually an error, though, as it suggests the prefs are corrupted somehow?  We could clarify this note in that case, but I'd be inclined to keep it as an error since it's likely either their doing or a busted Twinkle, yeah?

Reply:
Yes indeed!